backend/api/Areas/Leases/Mapping/Lease/PropertyMap.cs

```diff
@@ -9,30 +9,33 @@ public class PropertyMap : IRegister
     {
         public void Register(TypeAdapterConfig config)
         {
-            config.NewConfig<Entity.PimsProperty, Model.PropertyModel>()
+            config.NewConfig<Entity.PimsPropertyLease, Model.PropertyModel>()
+                .Map(dest => dest, src => src.Property)
                 .Map(dest => dest.Id, src => src.PropertyId)
-                .Map(dest => dest.PID, src => src.Pid)
-                .Map(dest => dest.PIN, src => src.Pin)
-                .Map(dest => dest.Name, src => src.Name)
-                .Map(dest => dest.AreaUnitId, src => src.PropertyAreaUnitTypeCode)
-                .Map(dest => dest.AreaUnit, src => src.PropertyAreaUnitTypeCodeNavigation.Description)
-                .Map(dest => dest.LandArea, src => src.LandArea)
-                .Map(dest => dest.Address, src => src.Address)
-                .Map(dest => dest.IsSensitive, src => src.IsSensitive)
-                .Map(dest => dest.Description, src => src.Description)
+                .Map(dest => dest.PID, src => src.Property.Pid)
+                .Map(dest => dest.PIN, src => src.Property.Pin)
+                .Map(dest => dest.Name, src => src.Property.Name)
+                .Map(dest => dest.AreaUnitId, src => src.Property.PropertyAreaUnitTypeCode)
+                .Map(dest => dest.AreaUnit, src => src.Property.PropertyAreaUnitTypeCodeNavigation.Description)
+                .Map(dest => dest.Address, src => src.Property.Address)
+                .Map(dest => dest.IsSensitive, src => src.Property.IsSensitive)
+                .Map(dest => dest.Description, src => src.Property.Description)
+                .Map(dest => dest.SurplusDeclaration, src => src)
                 .Map(dest => dest.RowVersion, src => src.ConcurrencyControlNumber)
-                .Map(dest => dest.SurplusDeclaration, src => src);
+                .Map(dest => dest.LandArea, src => src.LeaseArea);
 
             config.NewConfig<Model.PropertyModel,Entity.PimsPropertyLease>()
                 .Map(dest => dest.PropertyId, src => src.Id)
                 .Map(dest => dest.Property, src => src)
-                .Map(dest => dest.LeaseArea, src => src.AreaUnit)
-                .Map(dest => dest.Property, src => src.AreaUnitType.GetTypeId());
+                .Map(dest => dest.LeaseArea, src => src.LandArea)
+                .Map(dest => dest.Property, src => src.AreaUnitType.GetTypeId())
+                .Map(dest => dest.ConcurrencyControlNumber, src => src.RowVersion);
 
             config.NewConfig<Model.PropertyModel, Entity.PimsProperty> ()
                 .Map(dest => dest.PropertyId, src => src.Id)
                 .Map(dest => dest.Pid, src => src.PID == "" ? "-1" : src.PID)
```

Comment:
why do we need to use "-1" here? is it because PID is not nullable in the database?

Reply:
presumably? not part of this PR.